src/org/usfirst/frc/team2473/robot/commands/DriveForDistanceCommand.java

```diff
@@ -0,0 +1,98 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package org.usfirst.frc.team2473.robot.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.usfirst.frc.team2473.robot.Robot;
+import org.usfirst.frc.team2473.robot.RobotMap;
+import org.usfirst.frc.team2473.robot.subsystems.DrivePower;
+import org.usfirst.frc.team2473.robot.subsystems.DriveSubsystem;
+
+public class DriveForDistanceCommand extends Command {
+	
+	private static DrivePower SLOW_POWER = new DrivePower(0.1, 0.1, 0.1, 0.1);
+	
+	private double inches;
+	private double ticks; // should not be set directly
+	
+	private int prevTicks;
```

Comment:
Remember to document variables. I was confused as to what the heck previous ticks were